### DISCUSSION TITLE
请问rootfs怎么定制

### DISCUSSION THREAD

**Discussion by pokitpeng** [labring/sealos#4090] [Q&A]
如题，希望在gpu节点部署nvidia-docker，不知道如何定制rootfs

**Comment by cuisongliu** [comment]
> 如题，希望在gpu节点部署nvidia-docker，不知道如何定制rootfs

直接用gpu-operator

**Reply by pokitpeng** [reply]
多谢，我研究一下

**Comment by pokitpeng** [comment]
btw, 如果真的有定制rootfs的需求，官方会考虑出文档吗？
